packages/react/src/ActionList/ActionList.module.css

```diff
@@ -453,12 +453,6 @@
     & .ExpandIcon {
       transform: scaleY(-1);
     }
-
-    &.ActionListContent--hasActiveSubItem {
-      & > .ItemLabel {
-        font-weight: var(--base-text-weight-semibold);
-      }
-    }
```

Comment:
@langermank I did not see any usage of this class anywhere, so I removed it. Can you confirm?

Reply:
Oh weird, it is used in `Item` 

<img width="669" alt="AL item" src="https://github.com/user-attachments/assets/5c5615ac-05a9-49c9-8d5e-7d21796c0293" />

Reply:
Ah! Yes, that doesn't appear to be used.